tests/JsonValidatorTest.php

```diff
@@ -183,28 +183,6 @@ public function testValidateFailsIfSchemaNeitherStringNorObject()
     /**
      * @expectedException \Webmozart\Json\InvalidSchemaException
      */
-    public function testValidateFailsIfSchemaFileContainsNoObject()
```

Comment:
`get_object_vars() expects parameter 1 to be object, string given` in `justinrainbow/json-schema/src/JsonSchema/SchemaStorage.php:103`. Don't think there's a way to fix it here.